package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.1.0",
   "description": "A Strapi application",
   "scripts": {
+    "dev": "strapi develop",
     "develop": "strapi develop",
```

Comment:
Twice?

Reply:
yep, I will remove `develop`. It's just that they add `develop` by default, but all our projects use `dev` . I will remove `develop`